### PR TITLE
Add layer_norm doc to illustrate param_attr and bias_attr

### PR DESCRIPTION
**Pull request by sneaxiy**

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -2324,9 +2324,15 @@ def layer_norm(input,
...
+            gain :math:`g`. If :attr:`scale` is False, :attr:`param_attr` is
+            omitted. If :attr:`scale` is True and :attr:`param_attr` is None,
+            a default :code:`ParamAttr` would be added as scale. 
+            The :attr:`param_attr` is initialized as 1 if it is added.  
         bias_attr(ParamAttr|None): The parameter attribute for the learnable
-            bias :math:`b`.
+            bias :math:`b`. If :attr:`shift` is False, :attr:`bias_attr` is
+            omitted. If :attr:`shift` is True and :attr:`param_attr` is None,
+            a default :code:`ParamAttr` would be added as bias. 
+            The :attr:`bias_attr` is initialized as 0 if it is added.
```
Comment (chengduoZH) [inline python/paddle/fluid/layers/nn.py]:
```
... If :attr:`shift` is True and :attr:`param_attr` is None, a default :code:`ParamAttr` would be added as bias, and the default :attr:`bias_attr` is initialized as 0 if it is added. Default None.
```

Comment (chengduoZH) [inline python/paddle/fluid/layers/nn.py]: `
... If :attr:`shift` is True and :attr:`param_attr` is None, a default :code:`ParamAttr` would be added as bias, and the default :attr:`bias_attr` is initialized as 0 if it is added. Default None.
`

Comment (sneaxiy) [reply python/paddle/fluid/layers/nn.py]: Done.

**Review by chengduoZH — APPROVED** [review]
LGTM

**Pull request merged**